package.json

```diff
@@ -2053,9 +2053,9 @@
         ],
         "languageModelTools": [
             {
-                "name": "notebook_list_packages_tool",
+                "name": "notebook_list_packages",
                 "displayName": "List Notebook Kernel Packages",
-                "modelDescription": "List the installed packages that are currently available in the selected kernel for a notebook editor.",
+                "modelDescription": "List the installed packages that are currently available in the selected kernel for a notebook editor. This tool should be used when working with a notebook but only if already working in a notebook file.",
```

Comment:
@amunger you might want to update the message to indicate this ONLY applies to `Jupyter Notebooks`, we don't want this being used for other notebooks. Just like this got used for non-notebook files like Python files.

Reply:
> but only if already working in a notebook file

Reply:
but that doesn't mean that  applies to Jupyter notebooks, 
I.e. i could be working on a kusto notebook file or the like.

Reply:
Similarly, if we have packages missing in nodejs notebooks, then I don't think the model will know not to use this tool